### ISSUE TITLE
Multiple generators are broken in 0.9.9

### ISSUE THREAD

**Issue by pfrenssen** [hechoendrupal/DrupalConsole#1446]
It is no longer possible to use `generate:form`. It seems to be broken on many fronts in 0.9.9. When I roll back to 0.9.8 it works fine. Some of the symptoms I'm seeing:

* The questions are separated with two newlines, of which the first starts with a '>' symbol.
* Suggestions are no longer shown for the *Type*.
* Error messages are shown when trying to skip optional items like *Weight*.
* It's no longer possible to exit the loop, it keeps asking for new elements indefinitely.

**Comment by pfrenssen** [comment]
It is caused by https://github.com/hechoendrupal/DrupalConsole/commit/b7597874bbc66c8db0c6460ae357b71536c87ffc

**Comment by pfrenssen** [comment]
It was this issue: https://github.com/hechoendrupal/DrupalConsole/issues/575 - a whole bunch of files were changed in one single commit, so it's not possible to selectively revert part of it to unbreak this. We might need to revert the whole thing, or fix the problems one by one.

I just noticed that generate:plugin:block is also broken. It throws an error when you skip the optional *Header* question. I didn't check any of the other generators yet.

**Comment by jmolivas** [comment]
Looking at this.

**Comment by jmolivas** [comment]
I am currently testing the generators, and will be fixing issues.

**Comment by pfrenssen** [comment]
Cool, thanks!

**Comment by bojanz** [comment]
Happens in site:install as well.

**Comment by proxiss** [comment]
generate:entity:content is buggy now, Form files are created as src/FormXXXEntityForm.php insted of src/Entity/Form/XXXEntityForm.php.
Reference from annotation is therefore broken.

Thanks anyway!
Rainer Feike for proxiss

**Comment by kevinquillen** [comment]
I just installed drupal console fresh, and none of the generators are even in the list of commands for me.

So I tried:

```
drupal generate:module
```

and got:

```
[InvalidArgumentException]                                  
There are no commands defined in the "generate" namespace.
```

**Comment by jmolivas** [comment]
@kevinquillen: did you ran `drupal generate:module` from the root of your D8 site ?

You can find more information related to this topic at the doc pages => https://hechoendrupal.gitbooks.io/drupal-console/content/en/using/project.html

**Comment by kevinquillen** [comment]
Yes sir

**Comment by jmolivas** [comment]
@kevinquillen Is your site installed ?

**Comment by kevinquillen** [comment]
Yep

**Comment by jmolivas** [comment]
I am currently fixing the broken generators and the rest of the reported issues, expect a new release by Monday 14th.

**Comment by jmolivas** [comment]
@bojanz:
> Happens in site:install as well.

Fixed with #1472

**Comment by jmolivas** [comment]
@pfrenssen:
> The questions are separated with two newlines, of which the first starts with a '>' symbol.

This is because we are using [the new Symfony Console Style Guide](http://symfony.com/blog/new-in-symfony-2-8-console-style-guide).

> Suggestions are no longer shown for the Type.

You need to use the arrow keys (up/down), we defaulted to empty in order to be able to accept an empty value (since empty no longer not valid).
Improved with: #1473 

> Error messages are shown when trying to skip optional items like Weight.

Fixed with: #1473 

> It's no longer possible to exit the loop, it keeps asking for new elements indefinitely.

Fixed with: #1473   

> I just noticed that generate:plugin:block is also broken. It throws an error when you skip the optional Header question.

Fixed with:   #1474

**Comment by pfrenssen** [comment]
Great, thanks a lot!! I have just tested, and all the issues are fixed.

I'm totally fine with the two newlines and the '>' symbol, I just mentioned this because I thought it was a symptom of the bug.

Thanks again, you saved my D8 training I have to present next Friday!

**Comment by jmolivas** [comment]
@pfrenssen: I will be tagging a new release by today the later tomorrow. Fee free to ping mw with any other issues you found.

**Comment by jmolivas** [comment]
@proxiss:
> generate:entity:content is buggy now, Form files are created as src/FormXXXEntityForm.php insted of src/Entity/Form/XXXEntityForm.php.
Reference from annotation is therefore broken.

Fixed with #1475
